cnxdb/migrations/20170523212842_fast-in-book-search.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    # TODO migration code
```

Comment:
Can you remove the `TODO` item? These are nice to grep on and this. And some IDEs actually count these for you.